checkbox-ng/plainbox/impl/session/restart.py

```diff
@@ -279,6 +279,7 @@ def detect_restart_strategy(session=None, session_type=None) -> IRestartStrategy
         try:
             subprocess.run(
                 ['systemctl', 'is-active', '--quiet', 'checkbox-ng.service'],
+                env={"SYSTEMD_IGNORE_CHROOT": "1"},  # http://pad.lv/2003955
```

Comment:
This will clear other keys.
What I recommend doing instead is:
```
env = os.environ
env["SYSTEMD_IGNORE_CHROOT"]: "1"
subprocess.run( (...) env=env)
```

Reply:
Thanks, I've updated my patch